### DISCUSSION TITLE
Event logs

### DISCUSSION THREAD

**Discussion by azgard31** [dani-garcia/vaultwarden#2359] [General]
Will Event logs feature be implemented any soon?
Quite missing it

**Comment by BlackDex** [comment]
Event logs only work for ciphers and users in organizations and nowhere else.

I have a long-open-standing WIP branch which has a start, but not yet finished close.
It needs to be integrated into a lot of places, verified and checked which takes a lot of time. It will also add a maintenance burden i think.
I work on that feature sometimes, but not very often.

**Reply by GitGodius** [reply]
Hello, so how can i use it for Users in Organizations?
Thanks

**Reply by BlackDex** [reply]
At the moment you can't unless you build it from this PR #2868.
But, it is a WIP, and it could cause issues in between new commits, or database changes.
